omnibus/files/private-chef-cookbooks/private-chef/templates/default/oc_erchef.config.erb

```diff
@@ -56,6 +56,12 @@
         {error_logger_hwm, <%= @log_rotation['max_messages_per_second'] %>}
     ]},
 
+    {prometheus, [{collectors, [default,
+                                <% if node['private_chef']['postgresql']['enable'] && !node['private_chef']['postgresql']['external'] -%>
```

Comment:
Is there anything about the stats we collect that would make it hard to collect for external DBs?

Reply:
Not hard, but I was more thinking about who owns the service. The idea was, if the service is not on the box, it's not our problem to monitor. I could be convinced otherwise, however there might be weirdness if say multiple frontends were to report a service that they didn't own.